module/Frontpage/config/module.config.php

```diff
@@ -0,0 +1,43 @@
+<?php
+return array(
+    'controllers' => array(
+        'invokables' => array(
+            'Frontpage\Controller\Frontpage' => 'Frontpage\Controller\FrontpageController',
+        ),
+    ),
+    'router' => array(
+        'routes' => array(
+            'home' => array(
+                'type'    => 'Literal',
+                'options' => array(
+                    'route'    => '/',
+                    'defaults' => array(
+                        '__NAMESPACE__' => 'Frontpage\Controller',
+                        'controller'    => 'Frontpage',
+                        'action'        => 'home',
+                    ),
+                ),
+                'may_terminate' => true,
+                'child_routes' => array(
+                    'page' => array(
+                        'type'    => 'Segment',
+                        'options' => array(
+                            'route'    => '/[:category[/:name]]',
```

Comment:
This will be changed in the future once the page part is implemented. (Actually it is changed already on another branch.)